integration/src/main/java/com/pip/chatbot/integration/weather/ScheduledForecasts.java

```diff
@@ -33,6 +38,12 @@ public void saveForecasts() throws Exception {
             List<Forecast> forecasts = modelMapper.map(darkSkyApi.getWeatherForecast(city), new TypeToken<List<Forecast>>() {
             }.getType());
             forecastRepository.createForecasts(forecasts);
+
         }
     }
+
+    @Scheduled(fixedDelayString = "${application.darksky.forecastDelay}")
+    public void deleteOutdatedForecasts() {
+        forecastRepository.deleteOutdatedForecasts(LocalDateTime.now().minusDays(daysBeforeToRemove));
```

Comment:
Potrzebujemy to odzielnie schedulować? Ja bym po prostu wywoływał tą metodkę w saveForecasts, przed albo po pobraniu prognoz:)